**Review of Familial Natural Short Sleep (FNSS)**

### **Introduction**

Familial Natural Short Sleep (FNSS) is a rare, genetic, inherited trait that allows affected individuals to sleep 1-2 hours less each night with no downsides. This article reviews four published mutations that have been linked to FNSS, and presents several hypotheses of how FNSS safely reduces sleep. First, the two models on the restorative functions of sleep are recapped.

### **The Restorative Functions of Sleep**

#### **Synaptic Homeostasis[^1]**

* **Definition**: A form of homeostatic plasticity in which neuron synaptic strengths are adjusted up or down, but mostly down during sleep. Can co-occur with Hebbian plasticity.   
* **Mechanism:** During wake, Central Nervous System (CNS) sleep pressure is “tracked” based on their activity, and is selectively upscaled or downscaled during sleep. Homer1a binds to synapses with high activity to reduce its excitability, while Arc binds to inactive synapses and reduces its excitability further. Both processes involve coordination of numerous post-synaptic density (PSD) scaffolding proteins. Sleep pressure is tracked through molecular “counters” such as CaMKIIβ which have many phosphorylated states. Specifically, higher intracellular Ca2+ levels from synaptic activity increases the number of CaMKIIβ’s phosphorylated residues. Downstream, Homer1a and Arc bind conditionally based on CaMKIIβ phosphorylation levels, among other factors.   
* **Functional Importance**: Crucial for remembering, forgetting, scaling overall excitability to within safe ranges. 

#### **Waste Clearance[^2]**

* **Definition**: Neuron-coordinated clearance of metabolic waste (e.g., beta-amyloid, tau) clearance, primarily during deep NREM sleep.  
* **Mechanism**: Groups of neurons (up to 34\) were shown to fire in sync causing low-frequency, high-power ionic waves (Na+/K+). Interaction between high concentration of ions and H2O drives is hypothesized to drive bulk Cerebral Spinal Fluid (CSF) flow throughout the brain. Synced firing causes more powerful ionic waves, especially during slow wave sleep which occurs during non Rapid Eye Movement (NREM) sleep. Disruption of synchronized neural firing causes less powerful ionic waves and inhibits transport of metabolic waste out of the brain.  
* **Functional Importance**: Prevents toxic buildup, supporting neuronal health and reducing risks of neurodegenerative pathologies. CSF flow may have other functions besides waste clearance, and the synchronized oscillations may also have other functions.  

The recap is far from exhaustive. A more detailed review of the functions and mechanisms of sleep is presented by Zielinksi, et al[^3]. Readers who want a more concise version should refer to Pecurariu et al [^4].

### **Overview of FNSS-Related Gene Mutations**

The four main genes implicated—**ADRB1, NPSR1, DEC2, GRM1**—impact different molecular pathways but converge on a similar short-sleep phenotype without any apparent health deficits.

#### **ADRB1 (β1-Adrenergic Receptor) Mutation[^5] [^6]**

In Adrb1-A187V, alanine (A) is changed into valine (V) at position 187 of Beta-1 Adrenergic Receptor, a G-coupled protein receptor (GCPR). The alanine at position 187 is highly conserved invertebrates and invertebrates, suggesting functional importance. The mutation is very rare with a frequency of 4.028/100,000 according to the Exome Aggregation Consortium database.

1. **Mechanism of Action**   
   * **Reduced receptor stability** in β1-adrenergic receptor–expressing neurons.  
   * Hypothesized that Inhibitory function of β1-adrenergic receptor is more sensitive to reduction of its protein abundance while excitatory function is more tolerant to decreased protein levels – Dorsal Pons ADRB1+ neurons as a population becomes more active.   
   * The projection from central amygdala to locus coeruleus (LC) is more active, fostering higher baseline arousal.  
   * Enhanced receptor interaction with PSD-95 suggests more active synaptic signaling.   
2. **Human Study**  
   * Three affected individuals in the family had an average 2 hour reduction in sleep, out of a family of 8\.   
   * Autosomal dominant inheritance pattern observed.  
   * One carrier (\#100784) showed intermediate phenotype (7.5 hours sleep).  
   * Self-reported sleep schedules obtained through structured interviews.  
3. **Animal Study**  
   * **Sleep Architecture**  
     1. Mutant mice (Adrb1 \+/m) showed 55 minutes shorter sleep, with 53 minutes less NREM sleep and 7 minute REM sleep on average (N=13).  
     2. Mutant mice showed higher delta power  \- x1.5 normal vs. x1.3 in Wild Type (WT) mice \- at the beginning of light phase, but after the initial period (ZT4 and onwards) delta power was equal or slightly lower than WT mice.   
     3. ADRB1+ neurons (in both) showed increasing activity before NREM→REM and NREM→WAKE transitions, and decreased activity before WAKE→REM and REM→WAKE transitions.  
   * **Increased ADRB1+ neuron excitability**  
     1. Adrb1+/m were found to be more excitable. Mutant mice had a significantly lower rheobase current (\~10 pA) compared to wild-type mice (\~25 pA). Furthermore, there was \~10% decrease in number of inhibitory neurons, \~5% decrease in unchanged neurons, and no significant change in activation neurons in mutant mice vs. WT mice when treated with dobutamine and examining fluorescent traces.    
     2. Baseline spontaneous Excitatory Postsynaptic Current (sEPSC) frequency differences between mutant and WT mice were trending but not significant, but treatment with Dobutamine significantly increased sEPSC frequency in mutants but not WT mice.  
4. **Summary**  
   * Lower excitability thresholds and lower number of inhibitory ADRB1+ cells in Dorsal Pons (an arousal center) shifts Excitatory/Inhibitory balance to greater excitation in mutants, increasing arousal and wake time. Enhanced synaptic scaling or more powerful delta waves might explain restorative sleep despite reduced time.  
5. **Limitations**  
   * **Limited Human Data:** Single small family (n=8) with self-reported sleep data rather than objective measurements. Also, no validation in other populations or detailed health/cognitive assessments of carriers.   
   * **Mechanistic Gaps:** Complex interaction between excitatory/inhibitory circuits not fully mapped. Also, unclear how reduced receptor stability leads to selective loss of inhibitory function.   
   * **Translational Challenges:** Fundamental differences in sleep architecture between mice (polyphasic) and humans (consolidated) make direct translation of findings challenging. For example, mouse phenotype was much weaker than human (55min vs 2h less sleep). 

#### 

#### 

#### **NPSR1 (Neuropeptide S Receptor) Mutation[^7]**

Neuropeptide S receptor 1 is a GPCR with ligand neuropeptide S that has reported to modulate arousal and sleep behaviors[^8] [^9].  The mutation changes tyrosine (Y) into histidine (H) at position 206 in the protein sequence (Y206H), between the 4th and 5th transmembrane protein of NPSR1. It is located in one of the highly conserved across vertebrates, suggesting functional importance. The mutation is very rare with a frequency of 4.06x10^-6 in the Genome Aggregation database. 

1. **Mechanism of Action**  
   * Increased sensitivity of NPSR1 receptor leads to hyperactivity of sleep-wake regulating nuclei in Centromedian Thalamic Nucleus (CMT) and Lateral Hypothalamus (LH), and likely more wake-promoting regions.   
   * Maintaining arousal despite buildup of high sleep pressure. Sleep pressure is relieved through deeper NREM sleep, characterized by more powerful delta waves and more consolidated episodes.   
2. **Human Study**  
   * Family K50226 had two individuals with naturally short sleep duration (5.5 and 4.3 hours per night).  
   * Carriers were heterozygous for NPSR1-Y206H mutation.  
   * Sleep phenotype appears to be lifelong without obvious negative health effects  
   * Anecdotally, carriers maintained high energy and cognitive function despite reduced sleep.  
3. **Animal Study**  
   * **Sleep Architecture**  
     1. Mutant mice (NPsr1-Y026H KI) showed reduced total sleep time of 71 minutes compared to WT mice, across both light and dark phases.  
     2. Mutant mice showed significant reduction in NREM sleep during both dark and light phase, but only significant REM sleep reduction in dark phase. They also had fewer episodes of NREM bouts than WT mice (about 50 less, especially pronounced in the dark stage), showing a slightly correlated increase in NREM duration (but not statistically significant). REM bout/episode duration change were not significant[^10].  
   * **Arousal despite sleep pressure**  
     1. As evidenced by NREM delta power and SNIPPs (specific phosphorylated proteins) of EF2 and synapsin-1 levels in mutant mice, which were shown to increase during ZT2 and ZT22, but not at ZT11 when sleep pressure is lowest.   
     2. After 6 hours of sleep deprivation, mutant mice showed higher overall delta power during NREM sleep than WT mice, but interestingly it peaked during late deep sleep (ZT20). They also showed significantly less recovery sleep after SD (about 50 minutes less), requiring the same amount of recovery sleep in the first 6 hours (ZT6-12) but around 50 minutes less in the next 6 hours.   
   * **Increased NPSR1 receptor sensitivity and activity**  
     1. Mutant mice show higher levels of phosphorylated CREB (p-CREB), a key downstream effector of both Gs and Gq pathways, compared to wild-type, especially at ZT2 and ZT22.  
     2. When given NPS (1 n mol), mutant mice showed enhanced response (2x intensity), indicating the receptor is more sensitive to its ligand.   
     3. GCaMP signal response pattern in centromedial thalamus after NPS treatment showed mutant mice had a higher ratio of cells with group 2 (fast & long lasting) activation response to NPS (+6%).   
   * **Sleep deprivation has no effect on contextual memory**    
     1. Mutant mice in contextual fear memory tasks showed no loss of memory when given 6 hour SD treatment and tested again for memory, while WT mice showed memory loss.   
4. **Summary**  
   * Shift towards excitability in sleep-wake regulation areas allows for an awake state despite normal sleep pressure buildup, and deeper sleep is induced which performs all functions of sleep more efficiently.   
5. **Limitations**  
   * **Limited Human Data:** Only two human subjects from a single family, making it difficult to fully characterize the human phenotype and establish causality.  
   * **Mechanistic Gaps:** While increased receptor sensitivity was demonstrated, the complete pathway from enhanced NPSR1 signaling to reduced sleep need remains unclear. For example, the study focused mainly on CMT and LH regions, but NPSR1 is expressed throughout the brain \- broader effects are unknown  
   * **Long-term Effects:** No data on potential long-term consequences of chronically reduced sleep in either mice or humans, particularly regarding cognitive function, metabolism, and other physiological processes  
   * **Translation Challenges**

#### 

#### **DEC2 (BHLHE41) P384R Mutation**

DEC2 is a transcription factor that suppresses E-box-mediated transcription and regulates the circadian clock in mammals. The mutation changes proline (P) into arginine (R) at position 384 in the protein sequence (P384R). It is located in the Gly-Ala rich domain of DEC2, which mediates protein-protein interactions with other transcription factors like MyoD1. The mutation reduces DEC2's ability to suppress orexin expression by weakening its interaction with E12/47 transcription factors and reducing its DNA-binding affinity.

1. **Mechanism of Action**  
   * Transcriptional repressor DEC2 with diminished capacity to suppress orexin gene expression, resulting in elevated orexin levels. Specifically, weaker binding to Myod1/E12 TF complex, which binds to the E box site of prepro-orexin.   
   * Diminished capacity to inhibit circadian regulators (CLOCK/BMAL1)[^11].  
2. **Human Study**  
   * Small family study identified DEC2-P385R mutation in 2 carriers who naturally sleep \~6.25 hours vs. 8.06 hours for non-carriers.  
   * Carriers had similar sleep onset times (\~10pm) but much earlier wake times (\~4-4:30am) compared to non-carriers.  
   * Data was self-reported based on structured interviews about extended vacation periods.  
   * Total sample included 2 mutation carriers and 5 non-carrier family members.  
   * Sleep patterns were consistent across the lifespan according to subject reports.  
3. **Animal Study**  
   * **Sleep Architecture**  
     1. Embryonic DEC2P384R mice sleep 1–2 hours less, showing normal behavioral and cognitive outcomes.  
     2. Decreased % of NREM/REM sleep in Light phases only, no change in Dark.   
     3. NREM sleep duration significantly reduced (\~75 seconds/episodes in Light, \~30 seconds/episode in Dark), and NREM total episodes significantly increased (\~50 or more per Light), relative to WT mice.   
     4. No significant change in REM sleep duration, but slight decrease in REM sleep total episodes during Light period relative to WT mice.   
     5. Slightly increased average NREM delta power density (uV^2) across light and dark periods, but not in REM.   
   * **Recovery from Sleep Deprivation (6 hours)**  
     1. NREM sleep at Dark1: mutant mice showed only 17.1% increase vs. 71.8% increase in WT mice mice.  
     2. NREM sleep at Dark2: mutant mice showed only 2.0% increase vs 27.2% in WT mice.  
     3. REM sleep at Dark1**:** mutant mice showed 74.4% increase vs 175.2% in WT mice.  
     4. REM sleep @ Dark2: mutant mice showed 82.3% increase vs 122.4% in WT mice.  
     5. Mutant mice showed slower recovery despite starting with chronically less sleep, had significantly lower NREM delta power density changes after SD, while REM sleep differences were less pronounced and not statistically significant.   
   * **Orexin Antagonist Reverses FNSS phenotype**  
     1. Administration of orexin receptor antagonist MK-6096 (25 mg/kg) partially reversed FNSS phenotype in mice, as measured by immobility time over a 12 hour period after injection at ZT11.  
4. **Summary**  
   * Transcriptional-level changes (e.g., orexin elevation) shifts sleep–wake homeostasis, possibly allowing for arousal despite higher sleep pressure.    
5. **Limitations:**  
   * **Limited Human Data:** Single family study (n=7) with only two mutation carriers, relying solely on self-reported sleep patterns from structured interviews rather than objective measurements like polysomnography or actigraphy.  
   * **Mechanistic Understanding Gaps:** While the mutation's effects on transcriptional repression were demonstrated, the complete pathway from altered DEC2 function to reduced sleep need remains unclear. Furthermore, measurements from animal studies were limited to lateral hypothalamus.  
   * **Translation Challenges**

#### 

#### **GRM1 (mGluR1) Loss-of-Function Mutations[^12]**

Metabotropic glutamate receptor 1 (mGluR1/GRM1) is a GPCR that participates in synaptic plasticity and neurotransmission. Two different mutations were identified in unrelated families with natural short sleep: one mutation converts serine (S) to alanine (A) at position 458 (S458A) in the large N-terminal extracellular domain, while the other changes arginine (R) to tryptophan (W) at position 889 (R889W) in the C-terminal intracellular domain of the shorter mGluR1b isoform. Both positions are highly conserved across vertebrate species. Both mutations result in loss of receptor function, with reduced ERK activation and altered synaptic transmission. Both of the mutations are relatively rare in humans with a frequency of 2.52x10^-5 for GRM1bR889W and 1.59x10^-4 for GRM1-S458A.

1. **Mechanism of Action**   
   * **R889W** and **S458A** variants caused reduced Extracellular signal-Regulated Kinase (ERK) activation, leading to reduced sleep (increased signaling in ERK was found to lengthen sleep time).   
2. **Human Study**  
   * Two unrelated families (K50230 and K07331) identified with GRM1 mutations through whole exome sequencing  
   * Average actigraphy sleep of two affected individuals from one family (K50230) measured 4.6 hours of sleep and 4.1 hours of sleep over 16 days of recording.   
   * Maintenance of Wakefulness Test (MWT) showed the two FNSS individuals did not fall asleep in any nap, suggesting they aren’t sleep deprived    
   * Carriers maintain active lifestyle without apparent negative health consequences  
3. **Animal Study**:  
   * **Sleep Architecture**  
     1. 25-minute daily sleep reduction in mice models, decrease seen in NREM (P\<0.05) but not REM sleep.  
     2. Similar Delta power during NREM sleep to WT mice  
     3. Similar rebound after 5 hour sleep deprivation to WT mice  
   * **Electrophysiological Changes (in dentate gyrus neurons)**  
     1. Grm1-S458A showed more hyperpolarized resting membrane potential, decreased input resistance, and increased membrane capacitance  
     2. Grm1b-R889W maintained normal intrinsic membrane properties  
     3. Both mutations showed increased excitatory synaptic transmission in response to DHPG  
     4. Mutant cells showed altered mEPSC responses to DHPG compared to wild-type  
   * **Transcriptome Analysis**  
     1. Dentate gyrus neurons show heightened excitatory synaptic transmission.  
     2. Transcriptomic changes in ERK/Wnt pathways; normal sleep rebound and REM architecture.  
     3. 187 downregulated and 86 upregulated genes in Grm1b-R889W  
     4. 102 downregulated and 106 upregulated genes in Grm1-S458A  
     5. Significant overlap in gene expression changes between mutations  
     6. Few changes in known sleep-regulating genes  
4. **Summary**  
   * Glutamate receptor **loss-of-function** causes short sleep phenotype without changes in sleep pressure, rebound sleep.   
5. **Limitations**  
   * **Limited Human Data:** The study is based on only two families, with detailed sleep evaluation conducted on just two individuals from one family. Lacking comprehensive health/cognitive assessments and reliance on self-reported sleep data for most family members limits broader conclusions about the phenotype.   
   * **Mechanistic Gaps**: The precise mechanism linking GRM1 dysfunction to reduced sleep duration remains unclear. 

### 

### **Preliminary Hypotheses**

Here preliminary hypotheses are presented, with proposed experiments to validate it, and mutations to which it applies. 

*These are a work in progress and will likely be updated based on new data.* 

| Name | Premise | Experiment | Mutations |
| ----- | ----- | ----- | ----- |
| **Accelerated Synaptic Downscaling** | Mutations facilitate faster or more potent global synaptic upscaling and downscaling | Compare synaptic protein turnover rates during NREM delta waves between mutants and WT mice using time-resolved proteomics | ADRB1, NPSR1, GRM1 |
| **Accelerated Waste Clearance** | Stronger synchronized neuronal firing in short sleep episodes drivers more efficient CSF-ISF exchange and faster removal of metabolic waste | Measure CSF-ISF exchange rates using fluorescent tracers during sleep in mutants vs WT mice | All |

### **Conclusion**

Familial Natural Short Sleep (FNSS) suggests the possibility that core sleep functions—synaptic scaling and waste clearance—can be compressed without apparent negative impact on cognition or health. While ADRB1, NPSR1, DEC2, and GRM1 each interface with the sleep–wake axis through different molecular mechanisms, they converge on similar outcomes:

* ADRB1: A β1-adrenergic receptor mutation leading to heightened arousal states and more efficient synaptic tuning.

* NPSR1: A neuropeptide S receptor gain-of-function that elevates wakefulness yet preserves intense slow-wave bursts.

* DEC2: A transcription factor mutation that lifts orexin expression, modulating circadian and homeostatic drives for shorter but high-quality sleep.

* GRM1 (mGluR1): Loss-of-function variants in a metabotropic glutamate receptor, paradoxically increasing excitatory tone and abbreviating NREM.

Future work will involve systematically evaluating hypotheses to dissect whether each mutation shares final common pathways that underlie compressed sleep. Doing so will deepen our understanding of FNSS and potentially reveal new therapeutic avenues for optimizing sleep.

**References**

1. Diering GH. Remembering and forgetting in sleep: Selective synaptic plasticity during sleep driven by scaling factors Homer1a and Arc. Neurobiol Stress. 2022 Dec 31;22:100512. doi: 10.1016/j.ynstr.2022.100512. PMID: 36632309; PMCID: PMC9826981.  
2. Jiang-Xie LF, Drieu A, Bhasiin K, Quintero D, Smirnov I, Kipnis J. Neuronal dynamics direct cerebrospinal fluid perfusion and brain clearance. Nature. 2024 Mar;627(8002):157-164. doi: 10.1038/s41586-024-07108-6. Epub 2024 Feb 28\. PMID: 38418877\.  
3. Zielinski MR, McKenna JT, McCarley RW. Functions and Mechanisms of Sleep. AIMS Neurosci. 2016;3(1):67-104. doi: 10.3934/Neuroscience.2016.1.67. Epub 2016 Apr 21\. PMID: 28413828; PMCID: PMC5390528.  
4. Falup-Pecurariu C, Diaconu Ș, Țînț D, Falup-Pecurariu O. Neurobiology of sleep (Review). Exp Ther Med. 2021 Mar;21(3):272. doi: 10.3892/etm.2021.9703. Epub 2021 Jan 25\. PMID: 33603879; PMCID: PMC7851648.  
5. Shi G, Xing L, Wu D, Bhattacharyya BJ, Jones CR, McMahon T, Chong SYC, Chen JA, Coppola G, Geschwind D, Krystal A, Ptáček LJ, Fu YH. A Rare Mutation of β1-Adrenergic Receptor Affects Sleep/Wake Behaviors. Neuron. 2019 Sep 25;103(6):1044-1055.e7. doi: 10.1016/j.neuron.2019.07.026. Epub 2019 Aug 28\. PMID: 31473062; PMCID: PMC6763376.  
6. Dong Q, Ptáček LJ, Fu YH. Mutant β1-adrenergic receptor improves REM sleep and ameliorates tau accumulation in a mouse model of tauopathy. Proc Natl Acad Sci U S A. 2023 Apr 11;120(15):e2221686120. doi: 10.1073/pnas.2221686120. Epub 2023 Apr 4\. PMID: 37014857; PMCID: PMC10104526.  
7. Xing L, Shi G, Mostovoy Y, Gentry NW, Fan Z, McMahon TB, Kwok PY, Jones CR, Ptáček LJ, Fu YH. Mutant neuropeptide S receptor reduces sleep duration with preserved memory consolidation. Sci Transl Med. 2019 Oct 16;11(514):eaax2014. doi: 10.1126/scitranslmed.aax2014. PMID: 31619542; PMCID: PMC7587149.  
8. Reinscheid RK, Ruzza C. Pharmacology, Physiology and Genetics of the Neuropeptide S System. Pharmaceuticals (Basel). 2021 Apr 23;14(5):401. doi: 10.3390/ph14050401. PMID: 33922620; PMCID: PMC8146834.  
9. Xu YL, Reinscheid RK, Huitron-Resendiz S, Clark SD, Wang Z, Lin SH, Brucher FA, Zeng J, Ly NK, Henriksen SJ, de Lecea L, Civelli O. Neuropeptide S: a neuropeptide promoting arousal and anxiolytic-like effects. Neuron. 2004 Aug 19;43(4):487-97. doi: 10.1016/j.neuron.2004.08.005. PMID: 15312648\.  
10. [https://pmc.ncbi.nlm.nih.gov/articles/instance/7587149/bin/NIHMS1636587-supplement-NPSR1\_STM\_supp.pdf](https://pmc.ncbi.nlm.nih.gov/articles/instance/7587149/bin/NIHMS1636587-supplement-NPSR1_STM_supp.pdf)  
11. He Y, Jones CR, Fujiki N, Xu Y, Guo B, Holder JL Jr, Rossner MJ, Nishino S, Fu YH. The transcriptional repressor DEC2 regulates sleep length in mammals. Science. 2009 Aug 14;325(5942):866-70. doi: 10.1126/science.1174443. PMID: 19679812; PMCID: PMC2884988.  
12. Shi G, Yin C, Fan Z, Xing L, Mostovoy Y, Kwok PY, Ashbrook LH, Krystal AD, Ptáček LJ, Fu YH. Mutations in Metabotropic Glutamate Receptor 1 Contribute to Natural Short Sleep Trait. Curr Biol. 2021 Jan 11;31(1):13-24.e4. doi: 10.1016/j.cub.2020.09.071. Epub 2020 Oct 15\. PMID: 33065013\.

[^1]:  [https://www.sciencedirect.com/science/article/pii/S235228952200087X](https://www.sciencedirect.com/science/article/pii/S235228952200087X)

[^2]:  [https://www.nature.com/articles/s41586-024-07108-6](https://www.nature.com/articles/s41586-024-07108-6)

[^3]:  [https://pmc.ncbi.nlm.nih.gov/articles/PMC5390528/\#S21](https://pmc.ncbi.nlm.nih.gov/articles/PMC5390528/#S21)

[^4]:  [https://www.spandidos-publications.com/10.3892/etm.2021.9703](https://www.spandidos-publications.com/10.3892/etm.2021.9703)

[^5]:  [https://pmc.ncbi.nlm.nih.gov/articles/PMC6763376/](https://pmc.ncbi.nlm.nih.gov/articles/PMC6763376/)

[^6]:  [https://pmc.ncbi.nlm.nih.gov/articles/PMC10104526/](https://pmc.ncbi.nlm.nih.gov/articles/PMC10104526/)

[^7]:  [https://pmc.ncbi.nlm.nih.gov/articles/PMC7587149/](https://pmc.ncbi.nlm.nih.gov/articles/PMC7587149/)

[^8]:  [https://pubmed.ncbi.nlm.nih.gov/33922620/](https://pubmed.ncbi.nlm.nih.gov/33922620/)

[^9]:  [https://pubmed.ncbi.nlm.nih.gov/15312648/](https://pubmed.ncbi.nlm.nih.gov/15312648/)

[^10]:  Refer to Fig S4. C-F in [https://pmc.ncbi.nlm.nih.gov/articles/instance/7587149/bin/NIHMS1636587-supplement-NPSR1\_STM\_supp.pdf](https://pmc.ncbi.nlm.nih.gov/articles/instance/7587149/bin/NIHMS1636587-supplement-NPSR1_STM_supp.pdf)

[^11]:  [https://pubmed.ncbi.nlm.nih.gov/19679812/](https://pubmed.ncbi.nlm.nih.gov/19679812/)

[^12]:  [https://www.sciencedirect.com/science/article/pii/S096098222031441X](https://www.sciencedirect.com/science/article/pii/S096098222031441X)